CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+Unreleased
+----------
+
+**Fixed**
+
+* BadRequests that do not have any detailed error data are raised as
```

Comment:
Can you link to the exception?

Reply:
The issue that it fixes has the exception.